### PR TITLE
ADs' Contract Updates

### PR DESCRIPTION
**Pull request by blimpa**
This pull requests accumulates the contract preferences declared by ADs as per [this forum post](https://forum.makerdao.com/t/deprecation-of-ads-delegation-contract-2/24765). It must not be merged in before the deadline, 2024-08-10. For ADs who prefer to use contract 2, their contract 2 is moved onto the contract 1 column; therefore, when it's time to merge, column for contract 2 can be safely removed altogether.

### CODE REVIEW

File: MIP113/MIP113.md [hunk 1]
```diff
@@ -363,30 +363,30 @@ The list of all current recognized ADs is maintained in *6.1.2A*. Following the
...
+|---|---|---|---|
+| vigilant | [Address](https://etherscan.io/address/0x2474937cB55500601BCCE9f4cb0A0A72Dc226F61)<br>[Signature](https://etherscan.io/verifySig/16682) | [Contract](https://etherscan.io/address/0xDE08aEf2b221274231b3547491ec8f0Fc80917e1)<br>[Signature](https://etherscan.io/verifySig/255532) | [Link](https://forum.makerdao.com/t/cd-recognition-submission-vigilant/20457) |
+| QGov | [Address](https://etherscan.io/address/0xB0524D8707F76c681901b782372EbeD2d4bA28a6)<br>[Signature](https://etherscan.io/verifySig/16763) | [Contract](https://etherscan.io/address/0x0a907fe3adb890db7db27a7f21e188a4127b2e7c)<br>[Signature](https://etherscan.io/verifySig/16765) | [Link](https://forum.makerdao.com/t/qgov-cd-recognition-submission/20494) |
+| Bonapublica | [Address](https://etherscan.io/address/0x167c1a762B08D7e78dbF8f24e5C3f1Ab415021D3)<br>[Signature](https://etherscan.io/verifySig/21271) | [Contract](https://etherscan.io/address/0xe2bfDa5e1f59325E4b8dF5feaA30e4aB6516bf28)<br>[Signature](https://etherscan.io/verifySig/16766) | [Link](https://forum.makerdao.com/t/bonapublica-constitutional-delegate-recognition-submission/20451) |
```
Comment (0xldr) [inline MIP113/MIP113.md:370]: Contract has been updated source: https://forum.makerdao.com/t/bonapublica-aligned-delegate-communication/20451/90

```suggestion
| Bonapublica | [Address](https://etherscan.io/address/0x167c1a762B08D7e78dbF8f24e5C3f1Ab415021D3)<br>[Signature](https://etherscan.io/verifySig/21271) | [Contract](https://etherscan.io/address/0x452a7626f587d6811509dC071768168ea763EA3C)<br>[Signature](https://etherscan.io/verifySig/39657) | [Link](https://forum.makerdao.com/t/bonapublica-aligned-delegate-communication/20451/90) |
```

File: MIP113/MIP113.md [hunk 2]
```diff
@@ -363,30 +363,30 @@ The list of all current recognized ADs is maintained in *6.1.2A*. Following the
...
+| PBG | [Address](https://etherscan.io/address/0x8D4df847dB7FfE0B46AF084fE031F7691C6478c2)<br>[Signature](https://etherscan.io/verifySig/16773) | [Contract](https://etherscan.io/address/0x69b576a7e193a15a570ee5bb2149deb3f03537a2)<br>[Signature](https://etherscan.io/verifySig/21570) | [Link](https://forum.makerdao.com/t/pbg-constitutional-delegate-communication-platform/20471) |
```
Comment (0xldr) [inline MIP113/MIP113.md:371]: Contract has been updated. source: https://forum.makerdao.com/t/pbg-aligned-delegate-communication-platform/20471/69

```suggestion
| PBG | [Address](https://etherscan.io/address/0x8D4df847dB7FfE0B46AF084fE031F7691C6478c2)<br>[Signature](https://etherscan.io/verifySig/16773) | [Contract](https://etherscan.io/address/0xEb1764CC9577c52f430794E9044F64F1C49Dd395)<br>[Signature](https://etherscan.io/verifySig/40132) | [Link](https://forum.makerdao.com/t/pbg-aligned-delegate-communication-platform/20471/69) |
```

File: MIP113/MIP113.md [hunk 3]
```diff
@@ -363,30 +363,30 @@ The list of all current recognized ADs is maintained in *6.1.2A*. Following the
...
+| Pf | [Address](https://etherscan.io/address/0x94A388BEC0769c3394A48dB4DaBE5F29994100B3)<br>[Signature](https://etherscan.io/verifySig/16824) | [Contract](https://etherscan.io/address/0x62060879dfBB6dEf3B73CFC48f1f0595c0FeD505)<br>[Signature](https://etherscan.io/verifySig/16825) | [Link](https://forum.makerdao.com/t/pf-recognition-submission/20520) |
```
Comment (0xldr) [inline MIP113/MIP113.md:372]: Contract has expired

```suggestion
| Pf | [Address](https://etherscan.io/address/0x94A388BEC0769c3394A48dB4DaBE5F29994100B3)<br>[Signature](https://etherscan.io/verifySig/16824) | No active contract<br>[Signature](https://etherscan.io/verifySig/16825) | [Link](https://forum.makerdao.com/t/pf-recognition-submission/20520) |
```

File: MIP113/MIP113.md [hunk 4]
```diff
@@ -363,30 +363,30 @@ The list of all current recognized ADs is maintained in *6.1.2A*. Following the
...
+| UPMaker | [Address](https://etherscan.io/address/0xbb819df169670dc71a16f58f55956fe642cc6bcd)<br>[Signature](https://etherscan.io/verifySig/16997) | [Contract](https://etherscan.io/address/0xc241a035bc98128ac3983812a67dfc52ddeae09a)<br>[Signature](https://etherscan.io/verifySig/38946) | [Link](https://forum.makerdao.com/t/upmaker-cd-recognition-submission/20577) |
```
Comment (0xldr) [inline MIP113/MIP113.md:373]: Delegate asked to keep other contract (the one with MKR delegated to it). source: https://forum.makerdao.com/t/upmaker-ad-submission/20577/80

source for signature: https://forum.makerdao.com/t/upmaker-ad-submission/20577/59
```suggestion
| UPMaker | [Address](https://etherscan.io/address/0xbb819df169670dc71a16f58f55956fe642cc6bcd)<br>[Signature](https://etherscan.io/verifySig/16997) | [Contract](https://etherscan.io/address/0xc9C17EE58810172622b43B09C9C9FF31e50f5179)<br>[Signature](https://etherscan.io/verifySig/39034) | [Link](https://forum.makerdao.com/t/upmaker-cd-recognition-submission/20577) |
```

**Review by VoteWizard — APPROVED** [review]

File: MIP113/MIP113.md [hunk 5]
```diff
@@ -363,30 +363,30 @@ The list of all current recognized ADs is maintained in *6.1.2A*. Following the
...
+| WBC | [Address](https://etherscan.io/address/0xeBcE83e491947aDB1396Ee7E55d3c81414fB0D47)<br>[Signature](https://etherscan.io/verifySig/17927) | [Contract](https://etherscan.io/address/0x58dc40b5f9f0758ee43b0a208fe362c90fdfad4d)<br>[Signature](https://etherscan.io/verifySig/17929) | [Link](https://forum.makerdao.com/t/wbc-aligned-delegate-communications/20828) |
```
Comment (0xldr) [inline MIP113/MIP113.md:374]: New contract. source: https://forum.makerdao.com/t/wbc-aligned-delegate-communications/20828/76

```suggestion
| WBC | [Address](https://etherscan.io/address/0xeBcE83e491947aDB1396Ee7E55d3c81414fB0D47)<br>[Signature](https://etherscan.io/verifySig/17927) | [Contract](https://etherscan.io/address/0x3C85C3F4B0d610Fd29cc132A2073f85C15F83f2b)<br>[Signature](https://etherscan.io/verifySig/253745) | [Link](https://forum.makerdao.com/t/wbc-aligned-delegate-communications/20828) |
```

File: MIP113/MIP113.md [hunk 6]
```diff
@@ -363,30 +363,30 @@ The list of all current recognized ADs is maintained in *6.1.2A*. Following the
...
+| BLUE | [Address](https://etherscan.io/address/0xb6c09680d822f162449cdfb8248a7d3fc26ec9bf)<br>[Signature](https://etherscan.io/verifySig/19057) | [Contract](https://etherscan.io/address/0xb18680092734394295d0591bb42f2bd3c184517e)<br>[Signature](https://etherscan.io/verifySig/25260) | [Link](https://forum.makerdao.com/t/blue-ad-recognition-submission/20915) |
+| JAG | [Address](https://etherscan.io/address/0x58d1ec57e4294e4fe650d1cb12b96ae34349556f)<br>[Signature](https://etherscan.io/tx/0xcb017d31235abb7303c01169de358ffb1307c622994c802e1016dcdde8707164) | [Contract](https://etherscan.io/address/0x36124ceE63Ae786eea41Bda0FB61B1b946E4C08c)<br>[Signature](https://etherscan.io/tx/0xe195b455598d8ba6079af3654511474ce3b9c9cfeb5d7f942d3b80b3d5f83f20) | [Link](https://forum.makerdao.com/t/jag-ad-delegate-submission/20972) |
```
Comment (0xldr) [inline MIP113/MIP113.md:376]:
```suggestion
| JAG | [Address](https://etherscan.io/address/0x58d1ec57e4294e4fe650d1cb12b96ae34349556f)<br>[Signature](https://etherscan.io/tx/0xcb017d31235abb7303c01169de358ffb1307c622994c802e1016dcdde8707164) | No active contract<br>[Signature](https://etherscan.io/tx/0xe195b455598d8ba6079af3654511474ce3b9c9cfeb5d7f942d3b80b3d5f83f20) | [Link](https://forum.makerdao.com/t/jag-ad-delegate-submission/20972) |
```

File: MIP113/MIP113.md [hunk 7]
```diff
@@ -363,30 +363,30 @@ The list of all current recognized ADs is maintained in *6.1.2A*. Following the
...
+| Skynet | [Address](https://etherscan.io/address/0xd4d1A446cD5976a11bd32D3e815A9F85FED2F9F3)<br>[Signature](https://etherscan.io/verifySig/19702) | [Contract](https://etherscan.io/address/0x644c092a5ccafc425ebf133123299a6397ae97d9)<br>[Signature](https://etherscan.io/verifySig/19701) | [Link](https://forum.makerdao.com/t/skynet-ad-submission/21077) |
```
Comment (0xldr) [inline MIP113/MIP113.md:377]: new contract. source: https://forum.makerdao.com/t/skynet-ad-submission/21077/52

```suggestion
| Skynet | [Address](https://etherscan.io/address/0xd4d1A446cD5976a11bd32D3e815A9F85FED2F9F3)<br>[Signature](https://etherscan.io/verifySig/19702) | [Contract](https://etherscan.io/address/0x5c86fa06eebf5d7bdc1a75c293a3289ac0535f7f)<br>[Signature](https://etherscan.io/verifySig/251409) | [Link](https://forum.makerdao.com/t/skynet-ad-submission/21077) |
```

File: MIP113/MIP113.md [hunk 8]
```diff
@@ -363,30 +363,30 @@ The list of all current recognized ADs is maintained in *6.1.2A*. Following the
...
+| Cloaky | [Address](https://etherscan.io/address/0x869b6d5d8FA7f4FFdaCA4D23FFE0735c5eD1F818)<br>[Signature](https://etherscan.io/verifySig/19712) | [Contract](https://etherscan.io/address/0x475efaC48a0A18660A7A26eE6bd5FEBF466930f8)<br>[Signature](https://etherscan.io/verifySig/21627) | [Link](https://forum.makerdao.com/t/cloaky-ad-recognition-submission/21082) |
```
Comment (0xldr) [inline MIP113/MIP113.md:378]: New contract. source: https://forum.makerdao.com/t/cloaky-ad-recognition-submission/21082/60

```suggestion
| Cloaky | [Address](https://etherscan.io/address/0x869b6d5d8FA7f4FFdaCA4D23FFE0735c5eD1F818)<br>[Signature](https://etherscan.io/verifySig/19712) | [Contract](https://etherscan.io/address/0x24419b100Bd4f5C581c0AA89A274fcfB5B3915a8)<br>[Signature](https://etherscan.io/verifySig/250883) | [Link](https://forum.makerdao.com/t/cloaky-ad-recognition-submission/21082) |
```

File: MIP113/MIP113.md [hunk 9]
```diff
@@ -363,30 +363,30 @@ The list of all current recognized ADs is maintained in *6.1.2A*. Following the
...
+| Vision | [Address](https://etherscan.io/address/0xd21012e1fd245f6c25c0d527472fe42d2b78319f)<br>[Signature](https://etherscan.io/verifySig/23967) | [Contract](https://etherscan.io/address/0x3200c191Cc245b3E2de3FD3B3087104f3f313F57)<br>[Signature](https://etherscan.io/verifySig/23965) | [Link](https://forum.makerdao.com/t/vision-ad-recognition-submission/21777) |
+| Nimsen | [Address](https://etherscan.io/address/0x1331bFCc9496f2F73500a4dA1eAbEA324517889D)<br>[Signature](https://etherscan.io/verifySig/24053) | [Contract](https://etherscan.io/address/0xA2C669dc868be0A2a8D6C0ad715e17F45035BA76)<br>[Signature](https://etherscan.io/verifySig/24052) | [Link](http://forum.makerdao.com/t/nimsen-ad-recognition-submission/21794) |
+| Ikagai | [Address](https://etherscan.io/address/0x10078652eAB484c622030db7b4985BdccC626E99)<br>[Signature](https://etherscan.io/verifySig/24838) | [Contract](https://etherscan.io/address/0xAB3fD902E4dad36c6b31E3954A8dE14Dd829533c)<br>[Signature](https://etherscan.io/verifySig/24836) | [Link](https://forum.makerdao.com/t/ikagai-ad-recognition-submission/22124) |
```
Comment (0xldr) [inline MIP113/MIP113.md:382]:
```suggestion
| Ikagai | [Address](https://etherscan.io/address/0x10078652eAB484c622030db7b4985BdccC626E99)<br>[Signature](https://etherscan.io/verifySig/24838) | No active contract<br>[Signature](https://etherscan.io/verifySig/24836) | [Link](https://forum.makerdao.com/t/ikagai-ad-recognition-submission/22124) |
```

**Review by 0xldr — CHANGES_REQUESTED** [review]

**Pull request merged**